packages/now-static-build/build.sh

```diff
@@ -5,6 +5,6 @@ ncc build src/index.ts -o dist
 
 # copy gatsby plugin files in the gatsby redirect test case
```

Comment:
If the files are meant to be injected automatically, why does this build script need to copy them?

Reply:
The comment is wrong, it actually copy them into `dist` (similar to what we're doing with now-node-bridge). I will update the comment 👍